esmvalcore/cmor/_fixes/access/access_esm.py

```diff
@@ -0,0 +1,128 @@
+"""On-the-fly CMORizer for ACCESS-ESM.
+
+Note
+----
+This is the first version of ACCESS-ESM CMORizer in for ESMValCore
+Currently, only two variables (`tas`,`pr`) is fully supported.
+"""
+import logging
+import os
+
+from iris.cube import CubeList
+
+from esmvalcore.cmor._fixes.native_datasets import NativeDatasetFix
+
+logger = logging.getLogger(__name__)
+
+
+class Tas(NativeDatasetFix):
```

Comment:
Please put all of the functions that are the same for all variables (setting names, fixing the coord systems, etc.) in an `AllVars` fix. Almost everything you do here is already covered by [`NativeDatasetFix.fix_var_metadata`](https://github.com/ESMValGroup/ESMValCore/blob/ee16a958d916f1620a2aa25191e4cc34b3886006/esmvalcore/cmor/_fixes/native_datasets.py#L43-L83), so you can remove most of the and simply replace it with:

```py
class AllVars(NativeDatasetFix):
    """Fixes for all variables."""

    def fix_metadata(self, cubes):
        """Fix metadata."""
        cube = self.get_cube(cubes)

        # Fix scalar coordinates (you don't have this at the moment, but this might be helpful as well)
        self.fix_scalar_coords(cube)

        # Fix metadata of variable
        self.fix_var_metadata(cube)

        # Fix coord system
        self.fix_coord_system()

        return CubeList([cube])
```